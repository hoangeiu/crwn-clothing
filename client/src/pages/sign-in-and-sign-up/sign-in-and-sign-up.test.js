import React from "react";
import { shallow } from "enzyme";

import SignInAndSignUpPage from "./sign-in-and-sign-up.component";

it("should SignInAndSignUpPage component render without error", () => {
  expect(shallow(<SignInAndSignUpPage />)).toMatchSnapshot();
});
