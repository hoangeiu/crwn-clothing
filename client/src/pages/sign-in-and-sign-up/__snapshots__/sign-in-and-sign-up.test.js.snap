// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should SignInAndSignUpPage component render without error 1`] = `
<div
  className="sign-in-and-sign-up"
>
  <Connect(SignIn) />
  <Connect(SignUp) />
</div>
`;
